### PR TITLE
fix: squad header bar responsiveness

### PR DESCRIPTION
**Pull request by sshanzel**
### Describe what this PR does
- Apologies for some of the changes, it was pretty tough to read through the elements of the `SquadHeaderBar` as there are many things going on, so I have decided to extract it to a new component.
- The actual changes that fixed the responsiveness issue are:
  - Line 32: https://github.com/dailydotdev/apps/pull/1640/files#diff-13db82f69e64251bf61f24b0ae6b2d856e78937dc7817993ed6053655599ad49R32
  - Line 43: https://github.com/dailydotdev/apps/pull/1640/files#diff-13db82f69e64251bf61f24b0ae6b2d856e78937dc7817993ed6053655599ad49R43
  - Line 50: https://github.com/dailydotdev/apps/pull/1640/files#diff-13db82f69e64251bf61f24b0ae6b2d856e78937dc7817993ed6053655599ad49R50

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1114 #done

### CODE REVIEW

File: packages/shared/src/components/squads/SquadHeaderBar.tsx [hunk 1]
```diff
@@ -27,76 +24,30 @@ export function SquadHeaderBar({
...
-        initialSquad: squad,
-      },
-    });
 
   return (
     <div
       {...props}
       className={classNames(
-        'flex flex-row flex-wrap items-center gap-4',
+        'flex flex-row flex-wrap justify-center items-center gap-4 w-full',
```
Comment (sshanzel) [inline packages/shared/src/components/squads/SquadHeaderBar.tsx:32]: Fixed making every element centered in the actions bar.

File: packages/shared/src/components/squads/SquadHeaderBar.tsx [hunk 2]
```diff
@@ -27,76 +24,30 @@ export function SquadHeaderBar({
...
-          className="mobileL:order-2 btn btn-secondary"
+          className="tablet:order-2 btn btn-secondary"
           icon={<MenuIcon size="medium" />}
           onClick={onMenuClick}
           aria-label="Squad options"
         />
       </SimpleTooltip>
       <Button
-        className="w-full mobileL:w-auto btn btn-primary"
+        className="w-full mobileL:max-w-[18rem] tablet:w-fit btn btn-primary"
```
Comment (sshanzel) [inline packages/shared/src/components/squads/SquadHeaderBar.tsx:50]: Fixed the wrapping issue.

**Review by rebelchris — APPROVED** [review]
Thanks Lee,
From a code glance this looks right.

**Comment by sshanzel** [comment]
@rebelchris I have attached numerous screenshots for all screen sizes and they didn't seem to got updated 😭 

Let me try to upload it again.

**Comment by sshanzel** [comment]
@rebelchris let me know what you think with the preview 🚢

**Pull request merged**